tests/php/general/test_class.jetpack.php

```diff
@@ -1129,6 +1129,98 @@ public function test_wp_getOptions_hook_in_place() {
 		$this->assertArrayHasKey( 'jetpack_version', $options );
 	}
 
+	/**
+	 * Tests whether add_configure_hook saves the priority in case there were no hooks at all.
+	 *
+	 * @covers Jetpack::add_configure_hook
+	 */
+	public function test_add_configure_hook_saves_priority_on_no_hooks_present() {
+		global $wp_filter;
+
+		foreach ( $wp_filter['plugins_loaded']->callbacks as $priority => $callbacks ) {
+			foreach ( $callbacks as $action ) {
+				remove_action( 'plugins_loaded', $action['function'], $priority );
+			}
+		}
+
+		$this->assertArrayNotHasKey( 'plugins_loaded', $wp_filter );
+
+		$jetpack = Jetpack::init();
+		$jetpack->add_configure_hook();
+
+		$this->assertArrayHasKey( 'plugins_loaded', $wp_filter );
+		$this->assertSame( 0, $jetpack->configure_hook_priority );
+	}
+
+	/**
+	 * Tests whether add_configure_hook adds the default priority hook.
+	 *
+	 * @covers Jetpack::add_configure_hook
+	 */
+	public function test_add_configure_hook_adds_default_hook() {
```

Comment:
It looks like a lot of hooks are added to plugins_loaded before the test is executed, so this doesn’t actually test the default priority hook. `test_add_configure_hook_saves_priority_on_no_hooks_present()` does test the default priority hook because all of the plugins_loaded hooks are removed before calling Jetpack::init().